pkg/doc/vc/status/csl/csl.go

```diff
@@ -23,7 +23,8 @@ import (
 )
 
 const (
-	vcContext = "https://www.w3.org/2018/credentials/v1"
+	vcContext               = "https://www.w3.org/2018/credentials/v1"
+	jsonWebSignature2020Ctx = "https://w3c-ccg.github.io/lds-jws2020/contexts/lds-jws2020-v1.json"
```

Comment:
this isn't the canonical context

it's: https://w3id.org/security/jws/v1

Reply:
done. change it to ttps://w3id.org/security/jws/v1

Reply:
Looks like it got reverted.